packages/php73.rb

```diff
@@ -53,90 +53,90 @@ def self.patch
-           '--with-gmp',
-           '--with-libzip',
-           '--with-mysqli',
-           '--with-openssl',
-           '--with-pcre-regex',
-           '--with-pdo-mysql',
-           '--with-pear',
-           '--with-readline',
-           '--with-tidy',
-           '--with-unixODBC',
-           '--with-xsl',
-           '--with-zlib'
+    system "env TMPDIR='#{CREW_PREFIX}/tmp' CFLAGS='-pipe -ltinfo' \
+      ./configure \
+       --prefix=#{CREW_PREFIX} \
+       --docdir=#{CREW_PREFIX}/doc \
+       --infodir=#{CREW_PREFIX}/info \
+       --libdir=#{CREW_LIB_PREFIX} \
+       --localstatedir=#{CREW_PREFIX}/tmp \
+       --mandir=#{CREW_MAN_PREFIX} \
+       --sbindir=#{CREW_PREFIX}/bin \
+       --with-config-file-path=#{CREW_PREFIX}/etc \
+       --with-libdir=#{ARCH_LIB} \
+       --with-kerberos=#{CREW_LIB_PREFIX} \
+       --enable-exif \
+       --enable-fpm \
+       --enable-ftp \
+       --enable-mbstring \
+       --enable-opcache \
+       --enable-pcntl \
+       --enable-shared \
+       --enable-shmop \
+       --enable-sockets \
+       --enable-zip \
+       --with-bz2 \
+       --with-curl \
+       --with-gd \
+       --with-gettext \
+       --with-gmp \
+       --with-libzip \
+       --with-mysqli \
+       --with-openssl \
+       --with-pcre-regex \
+       --with-pdo-mysql \
+       --with-pear \
+       --with-readline \
+       --with-tidy \
+       --with-unixODBC \
+       --with-xsl \
+       --with-zlib"
     system 'make'
   end
 
   def self.check
-    #system 'make', 'test'
+    # system 'make', 'test'
   end
 
   def self.install
     system "mkdir -p #{CREW_DEST_PREFIX}/log"
     system "mkdir -p #{CREW_DEST_PREFIX}/tmp/run"
-    system "make", "INSTALL_ROOT=#{CREW_DEST_DIR}", "install"
+    system 'make', "INSTALL_ROOT=#{CREW_DEST_DIR}", 'install'
     system "install -Dm644 php.ini-development #{CREW_DEST_PREFIX}/etc/php.ini"
     system "install -Dm755 sapi/fpm/init.d.php-fpm.in #{CREW_DEST_PREFIX}/etc/init.d/php-fpm"
     system "install -Dm644 sapi/fpm/php-fpm.conf.in #{CREW_DEST_PREFIX}/etc/php-fpm.conf"
     system "install -Dm644 sapi/fpm/www.conf.in #{CREW_DEST_PREFIX}/etc/php-fpm.d/www.conf"
     system "ln -s #{CREW_PREFIX}/etc/init.d/php-fpm #{CREW_DEST_PREFIX}/bin/php7-fpm"
 
     # clean up some files created under #{CREW_DEST_DIR}. check http://pear.php.net/bugs/bug.php?id=20383 for more details
-    system "mv", "#{CREW_DEST_DIR}/.depdb", "#{CREW_DEST_LIB_PREFIX}/php"
-    system "mv", "#{CREW_DEST_DIR}/.depdblock", "#{CREW_DEST_LIB_PREFIX}/php"
-    system "rm", "-rf", "#{CREW_DEST_DIR}/.channels", "#{CREW_DEST_DIR}/.filemap", "#{CREW_DEST_DIR}/.lock", "#{CREW_DEST_DIR}/.registry"
+    system 'mv', "#{CREW_DEST_DIR}/.depdb", "#{CREW_DEST_LIB_PREFIX}/php"
+    system 'mv', "#{CREW_DEST_DIR}/.depdblock", "#{CREW_DEST_LIB_PREFIX}/php"
+    system 'rm', '-rf', "#{CREW_DEST_DIR}/.channels", "#{CREW_DEST_DIR}/.filemap", "#{CREW_DEST_DIR}/.lock",
+           "#{CREW_DEST_DIR}/.registry"
```

Comment:
Rubyize?

Reply:
Are those files or directories? As I understand it FileUtils.rm doesn't do directories.

Reply:
`FileUtils.rm_rf ...`?

Reply:
Ah, I read the documentation wrong on this then.